shell/platform/android/io/flutter/embedding/engine/FlutterEngine.java

```diff
@@ -339,7 +339,9 @@ public FlutterEngine(
     this.pluginRegistry =
         new FlutterEngineConnectionRegistry(context.getApplicationContext(), this, flutterLoader);
 
-    if (automaticallyRegisterPlugins) {
+    // Only automatically reigster plugins if both constructor parameter and
+    // loaded AnroidManifest config turn this feature on.
```

Comment:
nit: comment contains two typo's:

```suggestion
    // Only automatically register plugins if both constructor parameter and
    // loaded AndroidManifest config turn this feature on.
```